src/Pages/AddonsConfigPage.js

```diff
@@ -329,6 +487,8 @@ const FormContext = () => {
 			values.SliderSOCDInputEnabled = parseInt(values.SliderSOCDInputEnabled);
 		if (!!values.PlayerNumAddonEnabled)
 			values.PlayerNumAddonEnabled = parseInt(values.PlayerNumAddonEnabled);
+		if (!!values.PS4ModeAddonEnabled)
```

Comment:
I'm probably missing nuances here, i think all these checks might be reduced to:
```javascript
Object.entries(values)
  .reduce((acc, [key, value]) => ({
    ...acc,
    [key]: !!value ? parseInt(value) : value
  }), {})
```

Probably not in the scope of this PR, I can look at it after this is merged!